test/spec/grammar/grammar.js

```diff
@@ -0,0 +1,139 @@
+const { expect } = require('chai');
+const { parseFromContent } = require('../../../lib/readers/jdl_reader');
+
+describe('Grammar tests', () => {
```

Comment:
I noticed most (but not all) tests are in files named `*_test.js`. 
It seems `spec.js` suffix is a bit more standard. Would you accept a PR renaming test files to use `spec.js` suffix?

Reply:
yes of course!